### ISSUE TITLE
Add isServiceRunning helper to the react-native-track-player API

### ISSUE THREAD

**Issue by mitchdowney** [DoubleSymmetry/react-native-track-player#1320]
**Is your feature request related to a problem? Please describe.**
In our app we've wrapped some of our RNTP API calls in an `isServiceRunning` helper method to make sure the player service is available. It'd be great if we could do this through a native RNTP API method instead of having to maintain a .patch file to accomplish this.

[Here's our sample .patch file](https://github.com/podverse/podverse-rn/blob/react-native-track-player%402.1.2/patches/react-native-track-player%2B2.1.0.patch#L26). As you'd see by the TODO I'm not sure our logic is even correct...

**Describe the solution you'd like**
An API helper method that we can check to tell us the player service is running/available.

**Describe alternatives you've considered**
I haven't been able to find an alternative within the API docs...

**Comment by dcvz** [comment]
@mitchdowney how do you use this flag in your code? In case you haven’t ran setup or have destroyed the player?

**Comment by mitchdowney** [comment]
I think we added it as an attempt to resolve a nasty Android crash...but apparently it never worked so maybe no value in the API method I proposed. The crash that has been haunting us a long time on Android only is "attempt to invoke virtual method ... on a null object reference".

I just referenced this in this thread https://github.com/DoubleSymmetry/react-native-track-player/issues/1193

Here is a copy of the bug report we get on Android. Most are variations of the "attempt to invoke..." bug. ![image](https://user-images.githubusercontent.com/2262919/139766967-52448895-b773-4b9d-8ecc-a67e1397bdbe.png)

I wonder if our problem with our .patch file is the method we tried returns true no matter what so it doesn't help? https://github.com/podverse/podverse-rn/blob/react-native-track-player%402.1.2/patches/react-native-track-player%2B2.1.0.patch#L28
